package.json

```diff
@@ -198,7 +198,7 @@
     },
     {
       "path": "./dist/lighthouse-dt-bundle.js",
-      "maxSize": "450 kB"
+      "maxSize": "455 kB"
```

Comment:
I'm tempted to try reverting this again now; when I run this locally I get a pass with the existing `450 kB` limit:

```
 PASS  ./dist/lighthouse-dt-bundle.js: 441.38KB < maxSize 455KB (gzip) 
```